src/Shared/NcdfUtil/hco_ncdf_mod.F90

```diff
@@ -413,6 +413,18 @@ SUBROUTINE NC_READ_TIME( fID,     nTime,        timeUnit, &
     ! Read calendar attribute
     IF ( PRESENT( timeCalendar ) ) THEN
        CALL NcGet_Var_Attributes( fId, v_name, 'calendar', timeCalendar )
+
+       ! Throw an error for climatological calendars without leap years
+       SELECT CASE( TRIM( v_name ) )
+          CASE( '360_day', '365_day', 'noleap' )
+             WRITE( 6, '(/,a)' ) REPEAT( '=', 79 )
+             WRITE( 6, '(a)' )                                               &
+             'HEMCO does not support 360_day, 365_day, and noleap calendars!'
```

Comment:
Add 366_day and all_leap. Maybe the message could include the list of supported calendars?

Reply:
Now fixed in commit ba24290.  Also the list of supported calendars is displayed.

Reply:
A further fix for a syntax error was made in 7e20b3d0.